drivers/netdev_ieee802154_submac/netdev_ieee802154_submac.c

```diff
@@ -196,7 +196,7 @@ static int _recv(netdev_t *netdev, void *buf, size_t len, void *info)
 
     if (info) {
         netdev_ieee802154_rx_info_t *netdev_rx_info = info;
-        netdev_rx_info->rssi = rx_info.rssi;
+        netdev_rx_info->rssi = rx_info.rssi + IEEE802154_RADIO_RSSI_OFFSET;
```

Comment:
Note I'm adjusting here because this is `int16_t rssi;       /**< RSSI of a received frame in dBm */`

Reply:
you can maybe use the helpers in https://github.com/RIOT-OS/RIOT/pull/15629